imap_processing/lo/l0/data_classes/science_counts.py

```diff
@@ -280,7 +306,25 @@ def _parse_section(self, binary_string, decompression, data_shape):
         return data_array.reshape(data_shape[0], data_shape[1])
 
     def _extract_binary(self, binary_string, section_length, bit_length, decompression):
-        """Extract and decompress science count binary data section."""
+        """
+        Extract and decompress science count binary data section.
+
+        Parameters
+        ----------
+        binary_string : str
+            Binary string.
+        section_length : int
+            Length of section.
+        bit_length : int
+            Length of the bit.
+        decompression : int
+            The decompressed integer.
+
+        Returns
+        -------
+        np.array
```

Comment:
```suggestion
        numpy.array
```
it needs to be the full `numpy` to work in the automatic docs